californium-core/src/main/java/org/eclipse/californium/core/network/EndpointManager.java

```diff
@@ -48,29 +52,38 @@
  * outbound CoAP requests.
  *
  * The EndpointManager contains the default endpoint for coap (on port 5683) and
- * coaps (CoAP over DTLS). When an application serves only as client but not
- * server it can just use the default endpoint to send requests. When the
+ * endpoint for other schemes (coaps, CoAP over DTLS, coap+tcp, COAP over TCP,
+ * coaps+tcp, COAPS over TCP,). When an application serves only as client but
+ * not server it can just use the default endpoint to send requests. When the
  * application sends a request by calling {@link Request#send()} the send method
  * sends itself over the default endpoint.
  * <p>
  * To make a server listen for requests on the default endpoint, call
- * <pre>{@code
- *  CoapServer server = new CoapServer();
- * }</pre>
+ * 
+ * <pre>
+ * {
+ * 	CoapServer server = new CoapServer();
+ * }
+ * </pre>
+ * 
  * or more explicit
- * <pre>{@code
- *  Endpoint endpoint = EndpointManager.getEndpointManager().getDefaultEndpoint();
- *  CoapServer server = new CoapServer();
- *  server.addEndpoint(endpoint);
- * }</pre>
+ * 
+ * <pre>
+ * {
+ * 	Endpoint endpoint = EndpointManager.getEndpointManager().getDefaultEndpoint();
+ * 	CoapServer server = new CoapServer();
+ * 	server.addEndpoint(endpoint);
+ * }
+ * </pre>
  */
 public class EndpointManager {
 
 	/** The logger */
-	private final static Logger LOGGER = Logger.getLogger(EndpointManager.class.getCanonicalName());
+	private final static Logger LOGGER = Logger.getLogger(EndpointManager.class
```

Comment:
this should be `private static final`

you seem to be using shorter lines than defined by our Eclipse code formatting rules. Are you using them (contained in project root folder)?